subsys/zigbee/lib/zigbee_logger_ep/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+zephyr_library()
+zephyr_library_sources(zigbee_logger_eprxzcl.c)
+
+zephyr_library_link_libraries(zboss)
```

Comment:
Again, I don't think you need to link libraries here.

Reply:
Same as before, linking zboss library required to include ZBOSS headers.